packages/coinjoin/src/client/round/selectRound.ts

```diff
@@ -230,10 +234,12 @@ export const selectRound = async (
     );
     if (accountCandidates.length < 1) {
         log('No suitable accounts');
+        setSessionPhase({ phase: SessionPhase.RetryingCoinSelection, accountKeys });
```

Comment:
what exactly do you mean? The condition there is `accountCandidates.length < 1`, it's for the whole round, hence all accounts, no? They cannot fail separately, so the round is not created

Reply:
i guess i was trying to explain scenario [described here](https://github.com/trezor/trezor-suite/pull/7057#discussion_r1048364321) when 1 account is not picked (because of reason) and second will be picked,

not picked account will not get any message at all (because candidates > 0)